onnxruntime/python/tools/transformers/models/t5/t5_helper.py

```diff
@@ -66,6 +66,7 @@ def load_model(
         device: torch.device,
         merge_encoder_and_decoder_init: bool = True,
         model_type: str = "t5",
+        state_dict_path: str = "",
```

Comment:
```suggestion
        state_dict_path: Optional[str] = None,
```

Reply:
Dismissing in favor of keeping the API consistent with whisper